views/movies/movie-details.hbs

```diff
@@ -0,0 +1,17 @@
+<h1>{{title}}</h1>
+
+<p>Genre: {{genre}}</p>
+
+<p>Plot: {{plot}}</p>
+
+{{#each cast}}
+
+<h1>{{name}}</h1>
+
+<p>Acts as a: {{occupation}}</p>
+
+<p>Catch Phrase: {{catchPhrase}}</p>
+
+{{/each}}
+
+<a href="/movies">Go back</a>
```

Comment:
Make sure you format the code! Much easier to read and looks more organised. Here use indentation for example inside `each`